sprites.py

```diff
@@ -352,7 +352,7 @@ def draw(self, cr=None):
                              self.rect[3])
                 cr.fill()
             else:
-                print('sprite.draw: source not a pixbuf ({})'.format(type(img)))
+                print('sprite.draw: source not pixbuf ({})'.format(type(img)))
```

Comment:
There was nothing wrong with the original error text; I don't think it should be changed in this way.